Sample/SwiftUI/UserInfoView.swift

```diff
@@ -23,10 +24,10 @@ struct UserInfoView: View {
                 .textFieldStyle(.roundedBorder)
             }
             .alert(isPresented: $isEmptyTextFieldAlertPresented) {
-                Alert(title: Text("Invalid input format"), message: Text("Fill least one field"))
+                Alert(title: Text("alert_invalid_input_title"), message: Text("alert_fill_at_least_one_field"))
```

Comment:
How does that work? No need to call `localized` on string keys? (`"alert_invalid_input_title".localized`)

Reply:
Yes, It is work by only adding the localization key 😲. Anyway, I added localization to make it readable.